src/main/java/com/ibm/crypto/plus/provider/ock/CCMCipher.java

```diff
@@ -37,12 +36,8 @@ public final class CCMCipher {
     static final int CCM_AUGMENTED_MODE = 768;
 
     static {
-        disableCCMAcceleration = AccessController.doPrivileged(new PrivilegedAction<Boolean>() {
-            public Boolean run() {
-                return ("true"
-                        .equalsIgnoreCase(System.getProperty(DISABLE_CCM_ACCELERATION, "false")));
-            }
-        });
+        disableCCMAcceleration = Boolean.parseBoolean(
+            System.getProperty("jdk.nativeCryptoTrace", "false"));
```

Comment:
Why change the variable name here? Seems you updated this to match what is in openjdk for openssl support within Semeru?

Reply:
My bad. Got the code to mirror what we do in Semeru, but forgot to fix the flag. Thanks for catching that.

Reply:
Looks good now.